hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieHFileRecordReader.java

```diff
@@ -47,7 +48,7 @@
   public HoodieHFileRecordReader(Configuration conf, InputSplit split, JobConf job) throws IOException {
     FileSplit fileSplit = (FileSplit) split;
     Path path = fileSplit.getPath();
-    reader = new HoodieHFileReader(conf, path, new CacheConfig(conf));
+    reader = new HoodieHFileReader(conf, path, new CacheConfig(conf), Option.empty());
```

Comment:
is it required to fetch table props and pass is right value for recordKeyField here ?

Reply:
will explore more on this.

Reply:
is this resolved?